Doc/library/sqlite3.rst

```diff
@@ -2358,6 +2346,89 @@ can be found in the `SQLite URI documentation`_.
 .. _SQLite URI documentation: https://www.sqlite.org/uri.html
 
 
+.. _sqlite3-howto-row-factory:
+
+How to create and use row factories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, :mod:`!sqlite3` represents each row as a :class:`tuple`.
+If a :class:`!tuple` does not suit your needs,
+you can use the :class:`sqlite3.Row` class
+or a custom :attr:`~Cursor.row_factory`.
+
+While :attr:`!row_factory` exists as an attribute both on the
+:class:`Cursor` and the :class:`Connection`,
+it is recommended to set :class:`Connection.row_factory`,
+so all cursors created from the connection will use the same row factory.
+
+:class:`!Row` provides indexed and case-insensitive named access to columns,
+with minimal memory overhead and performance impact over a :class:`!tuple`.
+To use :class:`!Row` as a row factory,
+simply assign it to the :attr:`Connection.row_factory` attribute:
+
+.. doctest::
+
+   >>> con = sqlite3.connect(":memory:")
+   >>> con.row_factory = sqlite3.Row
+
+Query results are now returned as :class:`!Row` instances:
```

Comment:
I initially misread `Query` as a verb, so maybe it could be rephrased as:
```suggestion
Queries will now return :class:`!Row` instances:
```
or
```suggestion
The results of the queries will now be :class:`!Row` instances:
```

Reply:
or
```suggestion
Results from queries are now returned as :class:`!Row` instances:
```

Reply:
or

```suggestion
Results from queries will now be returned as :class:`!Row` instances:
```

(Of the other options, I like @ezio-melotti 's first option the most, perhaps more than my own here as its shorter; I preferred @merwok 's though I preferred it with the future perfect (i.e. just my suggestion) as opposed to the present progressive, and IMO the second suggestion is hard for me to read and the definite articles seem out of place.)

Reply:
I also like Ezio's first suggestion. Perhaps even without "will":

```suggestion
Queries now return :class:`!Row` instances:
```

Also, while we're at it, possibly use "objects" instead of "instances":

```suggestion
Queries now return :class:`!Row` objects:
```